FF133 puts Document.caretPositionFromPoint() shadowroot to nightly (#36910)
mdn/content: files/en-us/mozilla/firefox/releases/133/index.md

#### Removals

> - The [`options.shadowRoots`](/en-US/docs/Web/API/Document/caretPositionFromPoint#shadowroots) argument for passing {{domxref("ShadowRoot")}} objects to the {{domxref('Document.caretPositionFromPoint()')}} method has been withdrawn from the release. It remains available in the nightly release and is expected to re-release in future. ([Firefox bug 1914596](https://bugzil.la/1914596)).

Reviewer: What do you think about this?

```suggestion
- The [`options.shadowRoots`](/en-US/docs/Web/API/Document/caretPositionFromPoint#shadowroots) argument for passing {{domxref("ShadowRoot")}} objects to the {{domxref('Document.caretPositionFromPoint()')}} method has been withdrawn from the release.
  This can be enabled by setting the `dom.shadowdom.new_caretPositionFromPoint_behavior.enabled` preference to `true`, which is only enabled in the Nightly release by default. ([Firefox bug 1914596](https://bugzil.la/1914596)).
```

Author: I don't think so. If they want to try it they can use Nightly - that's why BCD strips out the preferences once things are in nightly. Its also better because the notes age badly once those preference values are removed. (FWIW I would prefer the pref remained in BCD, but I'm not in charge).

More important though, what I wanted to capture here is that it is still in nightly and is not expected to be a permanent withdrawal.

Minor tweak
```suggestion
- The [`options.shadowRoots`](/en-US/docs/Web/API/Document/caretPositionFromPoint#shadowroots) argument for passing {{domxref("ShadowRoot")}} objects to the {{domxref('Document.caretPositionFromPoint()')}} method has been withdrawn from the release.
  The feature can be tested in the Nightly release and is expected to re-release in future. ([Firefox bug 1914596](https://bugzil.la/1914596)).
```

Reviewer: > If they want to try it they can use Nightly - that's why BCD strips out the preferences once things are in nightly.

Gotcha, makes sense.
